python/ccxt/base/exchange.py

```diff
@@ -766,6 +766,8 @@ def extract_params(string):
 
     @staticmethod
     def implode_params(string, params):
+        if not isinstance(params, dict):
```

Comment:
It fails in all languages, not just Python.

Reply:
And it will also fail if one of keys under `params` is a list (not the `params` itself).